### PR TITLE
OC-8239:WS CRF import creates duplicate event CRF if a different CRF …

### PR DESCRIPTION
**Pull request by Taoli2018**
…version already has been started for that event.

### CODE REVIEW

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java [hunk 1]
```diff
@@ -352,6 +362,23 @@ public SummaryStatsBean generateSummaryStatsBean(ODMContainer odmContainer, List
...
+                    if (eventCRFBean == null) {
+                    	eventCRFBean = eventCRFDAO.findByEventCrfID(studyEvent, crfVersion);
+
+                    	if(eventCRFBean != null) {
+                			  int newCRFVersionId = crfVersion.getId();
+                			  eventCRFBean.setCRFVersionId(newCRFVersionId);
+
+                    	}
+
+                    }
```
Comment (svadla-oc) [inline ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java:381]: fix indentation in this class

**Review by svadla-oc — CHANGES_REQUESTED** [review]

**Comment by svadla-oc** [comment]
create a PR for 3.17.x branch

File: ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java (hunk 1 above)
Comment (Taoli2018) [reply ws/src/main/java/org/akaza/openclinica/web/crfdata/ImportCRFDataService.java:380]: fixed

**Comment by Taoli2018** [comment]
please see https://github.com/OpenClinica/OpenClinica/pull/3473 for 3.17 PR

**Comment by svadla-oc** [comment]
Addressed in #3473

**Pull request closed without merging**